ui/src/app/pages/admin/admin-institution-edit.spec.tsx

```diff
@@ -44,39 +45,79 @@ describe('AdminInstitutionEditSpec', () => {
       .toContain('Display name must be 80 characters or less');
   });
 
-  it('should default DUA to Master', async() => {
+  it('should default DOMAINS to ADDRESSES in Registered Tier requirement', async() => {
     urlParamsStore.next({
       institutionId: 'Verily'
     });
     const wrapper = component();
     await waitOneTickAndUpdate(wrapper);
     expect(wrapper).toBeTruthy();
 
-    const agreementTypeDropDown = wrapper.find('[data-test-id="agreement-dropdown"]');
-    expect(agreementTypeDropDown.first().props().value).toBe(InstitutionMembershipRequirement.DOMAINS);
+    const rtAgreementTypeDropDown = wrapper.find('[data-test-id="rt-agreement-dropdown"]');
+
+    expect(rtAgreementTypeDropDown.first().props().value).toBe(InstitutionMembershipRequirement.DOMAINS);
+  });
+
+  it('should default DOMAINS to ADDRESSES in Controlled Tier requirement', async() => {
+    urlParamsStore.next({
+      institutionId: 'Verily'
+    });
+    const wrapper = component();
+    await waitOneTickAndUpdate(wrapper);
+    expect(wrapper).toBeTruthy();
+
+    const ctEnabledToggle = wrapper.find('[data-test-id="ct-enabled-switch"]').first().instance() as InputSwitch;
+    ctEnabledToggle.props.onChange({
+      originalEvent: undefined,
+      value: true,
+      target: {name: 'name', id: '', value: true}
+    });
+    await waitOneTickAndUpdate(wrapper);
+
+    const ctAgreementTypeDropDown = wrapper.find('[data-test-id="ct-agreement-dropdown"]');
+    expect(ctAgreementTypeDropDown.first().props().value).toBe(InstitutionMembershipRequirement.DOMAINS);
+  });
+
+  it('should hide/show controlled card when controlled tier disabled/enabled', async() => {
+    const wrapper = component();
+    await waitOneTickAndUpdate(wrapper);
+    expect(wrapper).toBeTruthy();
+    const testInput = fp.repeat(81, 'a');
```

Comment:
copy paste, not used

Reply:
this appears in several locations actually

Reply:
it is not used even in the place where I copy that code. Deleted